### ISSUE TITLE
Create user profiles based on questionnaire responses

### ISSUE THREAD

**Issue by xlai** [xlai/dosefinderhub#3]
- [x] Develop code to capture primary user input - user profile and non-method specific questions
- [x] Develop code to capture secondary user input - method specific data, initially only for CRM and 3+3
- [x] Validate user responses (error checking)
- [x] Store responses for further processing

**Comment by sdennett20** [comment]
In a meeting today we discussed and agreed to reallocate some of the tasks. The purpose of the questionnaire is now to: 

1. Find out the top 3 (eventually, but choosing between 3+3 and CRM for now) most suitable trial designs for the user based on their input (including user profile data and some non-method-specific trial data) and recommend these, and
2. Ask the user for further information to collect method-specific trial data to then be passed on to the main UI. They will go through this process 3 times (once for now) to complete the data required for each of the 3 recommended designs (1 for now).

The user profiles are to be evaluated and used part way through the data collection process in order to reduce the quantity of data required (ie, run and compare only the 3 most suitable methods rather than, say, all 10). The questionnaire truly ends when the third set of model-specific data is given at which point the user switches to the main UI. The checkboxes in this issue have been updated to reflect this progression.
